pages/common/paci.md

```diff
@@ -0,0 +1,23 @@
+# paci
+
+> Your friendly, lightweight and flexible package manager. Written in Python.
+
+- Install `paci` and configure it:
+
+`bash <(curl -s https://raw.githubusercontent.com/tradebyte/paci/master/.github/bin/install.sh)`
+
+- Search for a given package:
+
+`paci search {{package}}`
+
+- Install a package:
+
+`paci install {{package}}`
+
+- Update a package:
+
+`paci update {{package}}`
+
+- Update the list of available packages and versions (it's recommended to run this before other `paci` commands):
```

Comment:
If it is recommended to run this first, then we should keep it as the first example.

Reply:
That's true. :)